pkg/apis/serving/v1alpha1/route_validation_test.go

```diff
@@ -74,6 +74,25 @@ func TestRouteValidation(t *testing.T) {
 			},
 		},
 		want: nil,
+	}, {
+		name: "valid annotation",
+		r: &Route{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "valid",
+				Annotations: map[string]string{
+					"networking.knative.dev/disableAutoTLS": "true",
+				},
+			},
+			Spec: RouteSpec{
+				Traffic: []TrafficTarget{{
+					TrafficTarget: v1.TrafficTarget{
+						RevisionName: "foo",
+						Percent:      ptr.Int64(100),
+					},
+				}},
```

Comment:
you can probably omit this for the test.

Reply:
doesn't look like it; if i include empty Spec or Traffic fields the test fails due to that field missing

Reply:
Spec sure, but what about `TrafficTarget` might be...